src/wazuh_modules/vulnerability_detector/cpe_helper.json

```
@@ -1,8 +1,61 @@
 {
     "version": "1.0",
     "format_version": "1.0",
-    "update_date": "2019-10-02T10:56Z",
+    "update_date": "2023-04-21T11:43Z",
     "dictionary": [
+	    {
+            "target": "windows",
+            "source": {
+                "vendor": [
+                    "Microsoft Corporation"
+                ],
+                "product": [
+                    "^Microsoft ASP.NET Core ([0-9]\\.*[0-9]*\\.*[0-9]*)"
+                ],
+                "version": [
+                    "^Microsoft ASP.NET Core ([0-9]\\.*[0-9]*\\.*[0-9]*)"
+            ]
+            },
```

Again, be careful with the indentation, please.
```suggestion
        {
            "target": "windows",
            "source": {
                "vendor": [
                    "Microsoft Corporation"
                ],
                "product": [
                    "^Microsoft ASP.NET Core ([0-9]\\.*[0-9]*\\.*[0-9]*)"
                ],
                "version": [
                    "^Microsoft ASP.NET Core ([0-9]\\.*[0-9]*\\.*[0-9]*)"
                ]
            },
```